modules/data_release/php/data_release.class.inc

```diff
@@ -68,16 +68,8 @@ class Data_Release extends \NDB_Menu_Filter
                                'upload_date' => 'upload_date',
                               );
 
-        $user_ID   = $DB->pselectOne(
-            "SELECT ID FROM users WHERE userid=:UserID",
-            array('UserID' => $user->getUsername())
-        );
-        $superuser = $DB->pselectOne(
-            "SELECT true FROM users WHERE ID IN "
-            . "(SELECT userid FROM user_perm_rel WHERE "
-            . "permid=1 AND userid=:UserID)",
-            array('UserID' => $user_ID)
-        );
+        $userCanUpload = $user->hasPermission("data_release_upload");
+        $userCanEdit   = $user->hasPermission("data_release_edit_permissions");
```

Comment:
Maybe the permission should be called just "data_release_edit"? We already know it's a permission.

Reply:
Lol the "permission" is what u edit its not the editing permission. I.e. data_release module lets you control access to specific files which is the add permission functionality. I appendedn _pernission to the name because you might want a diff edit code for lets say changing the file name which would be edit_file_name

Reply:
I could maybe rename it edit_access ?

Reply:
@HenriRabalais @driusan you could also voice your opinion since you agree with @johnsaigle